src/plugins/data_source/server/plugin.ts

```diff
@@ -3,19 +3,26 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { first } from 'rxjs/operators';
+
 import { PluginInitializerContext, CoreSetup, CoreStart, Plugin, Logger } from 'src/core/server';
-import { dataSource, credential } from './saved_objects';
+import { dataSource, credential, CredentialSavedObjectsClientWrapper } from './saved_objects';
+import { DataSourcePluginConfigType } from '../config';
 
 import { DataSourcePluginSetup, DataSourcePluginStart } from './types';
 
+import { CryptographyClient } from './cryptography';
+
 export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourcePluginStart> {
   private readonly logger: Logger;
```

Comment:
the naming conversion is not consistence , other class use`_` as prefix

Reply:
updated for naming consistency